TestingLowerBounds/ForMathlib/MaxMinEqAbs.lean

```diff
@@ -0,0 +1,13 @@
+import Mathlib.Algebra.Lie.OfAssociative
```

Comment:
That import looks really weird. Is that the result of minimize_imports?

Reply:
Yes it was suggested by `#min_imports`. I thought that since this lemmas should go to Mathlib it did not matter much which imports we have in the current file.